### PR TITLE
Ecf 116309685 contact point reschedule

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

**Comment by eschlange** [comment]
Is the main purpose of this change to augment the data gathered around task modification?

File: spec/controllers/scheduled_task_reschedulings_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  let(:tasks) { double("NurseTasks").as_null_object }
+  let(:valid_params) do
+    {
+      explanation: "asdf",
+      scheduled_at: Time.zone.now,
+      old_scheduled_at: Time.zone.now
+    }
+  end
+
+  shared_examples "a bad request" do
```
Comment (eschlange) [inline spec/controllers/scheduled_task_reschedulings_controller_spec.rb]: Nice use of shared_examples.

**Comment by ericcf** [comment]
@eschlange Yes, previously rescheduling a task just meant changing the date. Now they want to capture each rescheduling event along with a reason for it and notes about it.

**Comment by eschlange** [comment]
Looks great. It seems much more robust in terms of capturing the necessary task data. Nice refactoring of existing code as well.

**Pull request merged**